lib/ical/design.js

```diff
@@ -746,6 +756,15 @@ ICAL.design = (function() {
     timestamp: icalValues['date-time'],
     "language-tag": {
       matches: /^[a-zA-Z0-9-]+$/ // Could go with a more strict regex here
+    },
+    "phone-number":{
+      fromICAL: function(aValue) {
+        return fmap(Array.from(aValue), c => c==='\\' ? undefined : c  ).join('');
```

Comment:
Can you run the linters? I suspect this might fail due to missing spaces around operators.

As a bonus I'd also appreciate some performance testing to see if the conversion to and from an array is faster or slower than using indexOf and friends. I seem to recall there were some perf issues with large data sets in the past on this.